UnityProject/Chemistry/Reagent.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Chemistry
+{
+	public class Reagent
+	{
+		public string Name { get; internal set; }
+	}
+}
```

Comment:
What's the story with this one? Chilling in `UnityProject/Chemistry`?

Reply:
I'm not sure. It was in Chemistry before. There is another "reagent" in Botany too.

Reply:
From what I can see, this version is not referenced by anything. I have removed it.